### PR TITLE
figure3 revamp

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR adjusts figure3

### CODE REVIEW

File: figures/3.figure3/figures/PBMC_montage.png
Comment (jenna-tomkinson) [inline figures/3.figure3/figures/PBMC_montage.png:1]: Nice job adding the row labels!

File: figures/3.figure3/figures/PBMC_figure3.png
Comment (jenna-tomkinson) [inline figures/3.figure3/figures/PBMC_figure3.png:1]: No comments! I really like how the UMAP was simplified and how the figure was condensed!

File: figures/3.figure3/scripts/2.add_scale_bars.py
Comment (jenna-tomkinson) [inline figures/3.figure3/scripts/2.add_scale_bars.py:1]: Just a general question, can you add scale bars in your ImageJ macro? Like when you output images, can the scale bar already be added or no?

File: figures/3.figure3/scripts/1.retrieve_image_montage.py [hunk 1]
```diff
@@ -416,284 +448,293 @@
...
+
+        im2 = cv2.imread(image_path2.as_posix(), cv2.IMREAD_UNCHANGED)
+
+        im3 = cv2.imread(image_path3.as_posix(), cv2.IMREAD_UNCHANGED)
+
+        im4 = cv2.imread(image_path4.as_posix(), cv2.IMREAD_UNCHANGED)
+
+        im5 = cv2.imread(image_path5.as_posix(), cv2.IMREAD_UNCHANGED)
+
+        # check for non-edge cells
```
Comment (jenna-tomkinson) [inline figures/3.figure3/scripts/1.retrieve_image_montage.py:518]: What does this code comment reference?

File: figures/3.figure3/scripts/3.assemble_montage.r [hunk 2]
```diff
@@ -336,6 +347,159 @@ list_of_images <- list(
...
+options(repr.plot.width = width, repr.plot.height = height)
+
+# blank
+blank <- (
+    ggplot()
+    + geom_text(aes(x = 0.5, y = 0.5, label = ""), size = text_size)
+    + theme_void()
+)
+
+# ggplot of just text
```
Comment (jenna-tomkinson) [inline figures/3.figure3/scripts/3.assemble_montage.r:486]: This is really cool and super useful to make just text to add to the plot!

**Review by jenna-tomkinson — APPROVED** [review]
Nice PR! I have added a few questions but overall this figure is great!

File: figures/3.figure3/scripts/2.add_scale_bars.py
Comment (MikeLippincott) [reply figures/3.figure3/scripts/2.add_scale_bars.py:1]: No, it requires 'human in the loop' so I decided to use python

**Pull request merged**